src/odbc/unittests/common.c

```diff
@@ -93,7 +94,18 @@ odbc_read_login_info(void)
 	if (!PWD && !read_login_info_base(&common_pwd, "PWD"))
 		return 1;
 
-	/* find our driver */
+	/* 
+     * Some of the ODBC tests "go in the front door", i.e. call
+     * SQLDriverConnect() which will look up an entry in ODBCINST.INI .
+     * In CONNECT and TIMEOUT3 they look up the entry FreeTDS.
```

Comment:
Capitalized names looks odd for me. Would `"connect"` suit too? I think `ODBCINST.INI` is fine.